mobify/astro-scaffold
# Side-by-Side install

This PR enables side-by-side install of a Preview-enabled build and non-preview enabled. This makes it possible to install an App Store version and a preview version. 

JIRA: N/A
Linked PRs: https://github.com/mobify/astro-scaffold/pull/148

## Changes
- Add "Preview" app icon and configure it to be used in the `PreviewEnabledRelease` build config
- Use the `PRODUCT_BUNDLE_IDENTIFIER` for the `CFBundleIdentifier` in the Info.plist
- Create a new user-defined build setting `BUNDLE_IDENTIFIER`
- Use the new `BUNDLE_IDENTIFIER` for `CFBundleDisplayName` in the Info.plist 

## How to test-drive this PR
- Run the app 
- Switch to use the `PreviewEnabledRelease` configuration and run again.
- Note that there are two Scaffold apps (one that has a ` *P` suffix.. for "preview")

## TODOS:
